pyproject.toml

```diff
@@ -113,7 +112,7 @@ convention = "google"
 
 [tool.pylint]
 
-py-version = "3.6"
```

Comment:
I think the comments should be on the "minimum version" rather than "maximal supported version" (regardless of how those settings are named).

https://github.com/pylint-dev/pylint-pytest/actions/runs/6637112680/job/18031541929?pr=22

I think this is the direct cause of some of the above issues